tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -460,6 +474,7 @@ func (az *AzureProvider) getIPAddress(ctx context.Context) (ipAddress *string, e
 // scale to multiple listeners depending on the use case. By default they listen via Http protocol.
 // TODO: Provide the CertificateURL for the winRMListeners, the url need to be grabbed from the Key Vault.
 func constructWinRMConfig() (winRMConfig *compute.WinRMConfiguration) {
+
```

Comment:
why this space?

Reply:
Also do we need this function considering we're creating WinRMConfig in a different way now?